lib/step_flow/workflows/workflows.ex

```diff
@@ -19,7 +19,7 @@ defmodule StepFlow.Workflows do
       [%Workflow{}, ...]
 
   """
-  def list_workflows(params \\ %{}) do
+  def list_workflows(%{"rights" => user_rights} = params) do
```

Comment:
same as for: `list_workflow_definitions`
let: `list_workflows(params \\ %{})`

Query `rights` only if the key is present